web_ui/package.json

```diff
@@ -37,6 +38,7 @@
     "dotenv": "^16.5.0",
     "filesize": "^10.1.6",
     "framer-motion": "^12.9.4",
+    "geojson": "^0.5.0",
```

Comment:
What's the use case of `geojson` versus `turf`? From their readme it looks like it is very specific to geo-data (e.g. latitude and longitude information), do we need this to be able to use turf?

Reply:
More context: https://turfjs.org/docs/getting-started#typescript . But perhaps i can add it to devDependencies?

Reply:
Hmm but if they're only types then could we skip installing it completely? To me it would be fine if we don't use these type explicitly as they won't be part of any type contract.